profile/templates/fab/profile.html

```diff
@@ -135,20 +135,21 @@ <h3 class="box-heading heading-medium black-text column-one-half">Company descri
 
             <div class="message-box action-required border-medium border-black background-white">
               <div class="grid-row">
-                <h3 class="box-heading heading-medium black-text column-one-half">Case studies</h3>
+                <h3 class="box-heading heading-medium black-text column-one-half">Business showcase</h3>
               </div>
               <div class="grid-row">
-                  <p class="mid-grey-text font-xsmall">Describe something significant your company has done that will be relevant to international buyers.</p>
+                  <p class="mid-grey-text font-xsmall">Add images and success case studies to promote your products and services.</p>
                   <ul id="case-studies">
                       {% for case_study in company.supplier_case_studies %}
                           <li>
                             <img src="{{ case_study.image_one }}" width="60" />
                             <span>{{ case_study.title }}</span>
+                            <a id="edit-image" href="{% url 'find-a-buyer-case-study-edit' id=case_study.pk step='basic' %}" class="link pencil-link">Edit</a>
```

Comment:
it'd be good to append `case_study.pk` to the `"Edit"` link, cos otherwise there will be multiple links with the same `ID`
```suggestion
                            <a id="edit-image-{%case_study.pk%}"" href="{% url 'find-a-buyer-case-study-edit' id=case_study.pk step='basic' %}" class="link pencil-link">Edit</a>
```